GithubJobsEnterpriseProject/ClientApp/src/components/Register.js

```diff
@@ -1,37 +1,93 @@
-import React, {useState,useEffect} from 'react'
+import React, { useState, useEffect } from 'react'
+import axios from 'axios';
+import Login from './Login';
 
 export default function Register() {
-    const [user, getUser] = useState([]);
+
+    const [username, setUsername] = useState([]);
+    const [password, setPassword] = useState([]);
+    const [passwordRe, setPasswordRe] = useState([]);
+    const [email, setEmail] = useState([]);
+    const [isValidUsername, setIsValidUsername] = useState(false);
+    const [isValidEmail, setIsValidEmail] = useState(false);
+    const [isCredentailsValid, setIsCredentailsValid] = useState(null);
+
+    const handleChange = (e) => {
+        if (e.target.name == "Username") {
+            if (e.target.value !== null && e.target.value.length < 6) {
```

Comment:
This code is repeated. See next if statement.. It could be extracted into its own method e.g.:
```js
function validateLength(value){
}
```

Reply:
This line then could be changed like follows:

```diff
-if (e.target.value !== null && e.target.value.length < 6)
+if(validateLength(e.target.value))
```